kubernetes-manifests/problem-patterns/credit-card-meltdown-cronjob.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: enable-credit-card-meltdown
+spec:
+  schedule: "00 20 * * *"
```

Comment:
Other problem is already set to run at this time, they should be able to run together but we'd rather keep the default timeframes separate for each problem